docs/apache-airflow/authoring-and-scheduling/timetable.rst

```diff
@@ -19,49 +19,49 @@
 Timetables
 ==========
 
-For DAGs with time-based schedules (as opposed to event-driven), the scheduling
-decisions are driven by its internal "timetable".  The timetable also
-determines the data interval and the logical date of each run created for the DAG.
+For DAGs with time-based schedules (as opposed to event-driven), the DAGs internal "timetable"
+drives scheduling.  The timetable also determines the data interval and the logical date of
+each run created for the DAG.
 
 DAGs scheduled with a cron expression or ``timedelta`` object are
 internally converted to always use a timetable.
 
 If a cron expression or ``timedelta`` is sufficient for your use case, you don't need
 to worry about writing a custom timetable because Airflow has default timetables that handle those cases.
 But for more complicated scheduling requirements,
-you may create your own timetable class and pass that to the DAG's ``schedule`` argument.
+you can create your own timetable class and pass that to the DAG's ``schedule`` argument.
 
-Here are some examples of when custom timetable implementations are useful:
+Some examples of when custom timetable implementations are useful:
 
-* Data intervals with "holes" between. (Instead of continuous, as both the cron
-  expression and ``timedelta`` schedules represent.)
-* Run tasks at different times each day. For example, an astronomer may find it
+* Data intervals with "holes" between intervals instead of a continuous interval, as both the cron
```

Comment:
I think introducing the interval here might be confusing.

Reply:
maybe link to an explanation of data_interval conceptually? If such a doc exists